.pre-commit-config.yaml

```diff
@@ -59,3 +59,13 @@ repos:
           - --install-types # See mirrors-mypy README.md
           - --non-interactive
         additional_dependencies: [types-requests]
+
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v3.1.0"
+    hooks:
+      - id: prettier
+        types_or: [toml, yaml]
+        exclude: |
+          (?x)^(
+              \.github/workflows/(build|directory_writer|project_euler|ruff).yml
+          )$
```

Comment:
So actually this would not create the change from 51a75c514816ac1c63c01b9f4b69ba3a1ad81835.